app/models/miq_expression/field.rb

```diff
@@ -29,11 +29,18 @@ def datetime?
     column_type == :datetime
   end
 
+  def reflections
+    klass = model
+    associations.collect do |association|
+      klass.reflection_with_virtual(association).tap { |reflection| klass = reflection.klass }
```

Comment:
Is this used by sql and ruby expressions?
If this is only used by sql, does it need to use `reflect_on_association`?